### PR TITLE
Enable configs in artik053/extra

### PR DESCRIPTION
**Pull request by junmin-kim**
Enable romfs, arastorage, dm, iotivity, iotjs and system libs

### CODE REVIEW

File: build/configs/artik053/extra/defconfig [hunk 1]
```diff
@@ -553,8 +548,9 @@ CONFIG_UART4_PARITY=0
 CONFIG_UART4_2STOP=0
 # CONFIG_UART4_IFLOWCONTROL is not set
 # CONFIG_UART4_OFLOWCONTROL is not set
+# CONFIG_SENSOR is not set
 # CONFIG_USBDEV is not set
-# CONFIG_FOTA_DRIVER is not set
+CONFIG_FOTA_DRIVER=y
```
Comment (sunghan-chang) [inline build/configs/artik053/extra/defconfig]: revert this

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by junmin-kim** [comment]
@sunghan-chang Please review this

**Comment by seinfra** [comment]
Code Rule Check OK.

File: build/configs/artik053/extra/defconfig [hunk 2]
```diff
@@ -915,7 +996,8 @@ CONFIG_PM=y
 # CONFIG_DEBUG_PM is not set
 # CONFIG_PM_TEST is not set
 CONFIG_PM_DEVNAME_LEN=32
-# CONFIG_PM_METRICS is not set
+CONFIG_PM_METRICS=y
```
Comment (sunghan-chang) [inline build/configs/artik053/extra/defconfig]: Disable PM

File: build/configs/artik053/extra/defconfig [hunk 3]
```diff
@@ -961,7 +1044,7 @@ CONFIG_DEBUG_NET=y
 # CONFIG_ARCH_HAVE_HEAPCHECK is not set
 CONFIG_DEBUG_MM_HEAPINFO=y
 # CONFIG_DEBUG_IRQ is not set
-# CONFIG_DEBUG_IRQ_INFO is not set
+CONFIG_DEBUG_IRQ_INFO=y
```
Comment (sunghan-chang) [inline build/configs/artik053/extra/defconfig]: Disable

File: build/configs/artik053/extra/defconfig [hunk 4]
```diff
@@ -1064,13 +1137,36 @@ CONFIG_NETDB_DNSSERVER_BY_DHCP=y
 #
 # External Libraries
 #
-# CONFIG_LWM2M_WAKAAMA is not set
-# CONFIG_DM is not set
-# CONFIG_ENABLE_IOTIVITY is not set
-# CONFIG_LIBTUV is not set
-# CONFIG_AWS_SDK is not set
+CONFIG_AWS_SDK=y
```
Comment (sunghan-chang) [inline build/configs/artik053/extra/defconfig]: Disable aws

File: build/configs/artik053/extra/defconfig [hunk 5]
```diff
@@ -1110,23 +1209,33 @@ CONFIG_EXAMPLES_HELLO=y
...
 CONFIG_EXAMPLES_SLSIWIFI_PRIORITY=50
 CONFIG_EXAMPLES_SLSIWIFI_STACKSIZE=2048
 # CONFIG_EXAMPLES_SMART is not set
 # CONFIG_EXAMPLES_SMART_TEST is not set
 # CONFIG_EXAMPLES_SMTPCLIENT is not set
-# CONFIG_EXAMPLES_SYSIO_TEST is not set
+# CONFIG_EXAMPLES_ST_THINGS is not set
+# CONFIG_EXAMPLES_TELNETD is not set
 # CONFIG_EXAMPLES_TESTCASE is not set
+CONFIG_DM_AP_SSID="SSID"
```
Comment (sunghan-chang) [inline build/configs/artik053/extra/defconfig]: I modified this. Please use latest. It should be under dm, not tc.

File: build/configs/artik053/extra/defconfig [hunk 6]
```diff
@@ -1174,12 +1297,34 @@ CONFIG_ENABLE_ENV_SET=y
 CONFIG_ENABLE_ENV_UNSET=y
 CONFIG_ENABLE_FREE=y
 CONFIG_ENABLE_HEAPINFO=y
-# CONFIG_ENABLE_IRQINFO is not set
+CONFIG_ENABLE_IRQINFO=y
```
Comment (sunghan-chang) [inline build/configs/artik053/extra/defconfig]: disable

File: build/configs/artik053/extra/defconfig [hunk 7]
```diff
@@ -844,7 +907,7 @@ CONFIG_FS_PROCFS=y
 # CONFIG_FS_PROCFS_EXCLUDE_PARTITIONS is not set
```
Comment (sunghan-chang) [inline build/configs/artik053/extra/defconfig]: enable proc

Comment (junmin-kim) [reply build/configs/artik053/extra/defconfig]: This config is `EXCLUDE` feature. Should I enable this?

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Pull request merged**